core/src/main/resources/reference.conf

```diff
@@ -54,29 +54,19 @@ akka.kafka.consumer {
 
   # If commits take longer than this time a warning is logged
   commit-time-warning = 1s
-  
-  # If for any reason `KafkaConsumer.poll` blocks for longer than the configured
-  # poll-timeout then it is forcefully woken up with `KafkaConsumer.wakeup`.
-  # See https://kafka.apache.org/10/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#wakeup--
-  # The KafkaConsumerActor will throw
-  # `org.apache.kafka.common.errors.WakeupException` which will be ignored
-  # until `max-wakeups` limit gets exceeded.
-  # DEPRECATED: No op as waking up consumer is not used anymore
-  wakeup-timeout = 3s
-
-  # After exceeding maximum wakeups the consumer will stop and the stage will fail.
-  # Setting it to 0 will let it ignore the wakeups and try to get the polling done forever.
-  # DEPRECATED: No op as waking up consumer is not used anymore
-  max-wakeups = 10
+
+  # Not used anymore (since 1.0-RC1)
+  # wakeup-timeout = 3s
```

Comment:
Should we just remove these altogether?

Reply:
My thinking was to leave them (during 1.0.x) so people looking at their existing configs understand that they may remove them.

Reply:
Alright.